### DISCUSSION TITLE
New version not returned when bump - alternatives ?

### DISCUSSION THREAD

**Discussion by BenderRules** [callowayproject/bump-my-version#192] [Q&A] [answered]
Hi,

This tool is working great, as part of our CI/CD pipelines. Unless I'm missing something though, I cannot see a way to know the new version once bumped. Is there a way to just get the version in the toml or cfg file, whether before or after a bump. Since it's not returned by the bump command itself, is there a command that allows that ? I checked the show commands but it returns the whole config file's content. Thanks in advance for any reply.

**UPDATE** : Ok, I just found out that I could simply do : _show current_version_. Case closed then. Still, could be nice to have the bump command return the new version afterwards.

**Comment by coordt** [comment, marked as answer]
Your suggestion has merit.

The behavior of *not* returning the version was implemented long before my fork of the project. As such, I hesitate to change a behavior that might break existing users' experience. That's why I implemented the range of *show* subcommands.

With all of that said, you can put in a feature request on the issue board to have a switch that will return the new version to stdout if you think it is an important feature.

**Reply by BenderRules** [reply]
Thanks for your reply! In fact, yes I found out that the show command easily returns whatever we need, including the version. I think perhaps just enhancind the doc for the show command would be sufficent, as it took me a little while to fully understand its flexibility.
